docs/conf.py

```diff
@@ -21,9 +24,11 @@
 sys.path.insert(0, os.path.abspath('../cla_backend/libs/'))
 sys.path.insert(0, os.path.abspath('..'))
 sys.path.insert(0, os.path.abspath('../../cla_docs/docs/source/'))
-from common_conf import *
+from common_conf import *  # cla_docs.source.common_conf
 
 extensions += ['docs.swag']
+# extensions += ['docs.swag', 'm2r']
+# source_suffix = ['.rst', '.md']
```

Comment:
🙋‍♂️ Do we need the comments in these two lines?

Reply:
Indeed we don't. They relate to a newly spawned ticket, but can better wait in a stash.